model_zoo/lbann.cpp

```diff
@@ -49,18 +49,13 @@ int main(int argc, char *argv[]) {
     std::cout << std::endl << std::endl;
   }
 
-#ifdef EL_USE_CUBLAS
-  El::GemmUseGPU(32,32,32);
-#endif
-
   try {
     // Initialize options db (this parses the command line)
     options *opts = options::get();
     opts->init(argc, argv);
     if (opts->has_string("h") or opts->has_string("help") or argc == 1) {
-      print_help(comm);
-      finalize(comm);
```

Comment:
`finalize` is not skipped. (That applies pre-edit)

Reply:
Agree re another PR. That would have been a change of semantics, which I was largely trying to avoid.

Punchline: `finalize()` is still called whenn `comm` is destroyed at the end of scope.